pkgs/development/compilers/scala/bare.nix

```diff
@@ -20,23 +30,27 @@ stdenv.mkDerivation rec {
   # Use preFixup instead of fixupPhase
   # because we want the default fixupPhase as well
   preFixup = ''
-        bin_files=$(find $out/bin -type f ! -name common)
-        for f in $bin_files ; do
-          wrapProgram $f --set JAVA_HOME ${jre} --prefix PATH : '${ncurses.dev}/bin'
-        done
+    bin_files=$(find $out/bin -type f ! -name "*common*" ! -name "scala-cli.jar")
+    for f in $bin_files ; do
+      wrapProgram $f --set JAVA_HOME ${jre} --prefix PATH : '${ncurses.dev}/bin'
+    done
   '';
 
   meta = with lib; {
     description = "Research platform for new language concepts and compiler technologies for Scala";
     longDescription = ''
-       Dotty is a platform to try out new language concepts and compiler technologies for Scala.
-       The focus is mainly on simplification. We remove extraneous syntax (e.g. no XML literals),
-       and try to boil down Scala’s types into a smaller set of more fundamental constructs.
-       The theory behind these constructs is researched in DOT, a calculus for dependent object types.
+      Dotty is a platform to try out new language concepts and compiler technologies for Scala.
+      The focus is mainly on simplification. We remove extraneous syntax (e.g. no XML literals),
+      and try to boil down Scala’s types into a smaller set of more fundamental constructs.
+      The theory behind these constructs is researched in DOT, a calculus for dependent object types.
```

Comment:
I think we should no longer advertise Dotty as an experimental platform, as it is now *the* Scala 3 compiler. Simplifying it to just

> The Scala 3 compiler, also known as Dotty.

Should work imo.